cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -105,7 +105,7 @@
             <div class="o-post-preview_image-container">
                 <img class="o-post-preview_image"
                      src="{{ photo.url }}"
-                     alt="{{ post.preview_image.alt if post.preview_image.alt else '' }}">
+                     alt="{{ photo.alt }}">
```

Comment:
Hmm... I suppose we could do that if we altered the function to accept either an ImageBasic object or nothing at all. What do you think about doing that, @chosak?